pkg/commands/commands.go

```diff
@@ -340,6 +344,37 @@ func GolangCI(fix bool) error {
 	)
 }
 
+
+func ReviewDog(PR int) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported function ReviewDog should have comment or be unexported